Workbooks/CyberArkEPV.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
The png screenshots shared for light and dark backgrounds do not match with this workbook - this one has a table and then graphs in a row, while the PNG files look very compact. Is it possible to match the workbook (json) layout of this workbook to match the PNG look (in which case you might want to have the note in a block of it's own, to avoid interfering with the look of the workbook.